dashboard/index.html

```diff
@@ -0,0 +1,122 @@
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" >Rezpozytoria
+                        <span class="caret"></span>
+                    </a>
+                    <ul class="dropdown-menu" role="menu" >
+                        <li><a href="https://github.com/infoshareacademy/jfdzw1-projekty">projekty</a></li>
+                        <li><a href="https://github.com/infoshareacademy/jfdzw1-materialy">materiały</a></li>
+                        <li><a href="https://github.com/infoshareacademy/jfdzw1-zadania-domowe">zadania domowe</a></li>
+                    </ul>
+                </li>
+        </ul>
+    </div>
+</nav>
+<div class="container">
+    <div class="row">
+        <div class="col-md-9 col-lg-9">
+            <div id="imageCarousel" class="carousel slide" data-interval="3000" data-ride="carousel">
+             <ol class="carousel-indicators">
+                 <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+                 <li data-target="#carousel-example-generic" data-slide-to="1"></li>
+                 <li data-target="#carousel-example-generic" data-slide-to="2"></li>
+             </ol>
+         <div class="carousel-inner">
+             <div class="item active">
+                 <img src="img/motto.jpg" alt="motto">
+                 <div class="carousel-caption">
+             </div>
+            </div>
+            <div class="item">
+                <img src="img/265.jpg" alt="motivation photo">
+                <div class="carousel-caption">
+                </div>
+            </div>
+                <div class="item">
+                    <img src="img/motivational.png" alt="second motivation photo">
+                    <div class="carousel-caption">
+                    </div>
+                </div>
+            <!-- Controls -->
+            <a class="left carousel-control" href="#carousel-example-generic" role="button" data-slide="prev">
+                <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+                <span class="sr-only">Previous</span>
+            </a>
+            <a class="right carousel-control" href="#carousel-example-generic" role="button" data-slide="next">
+                <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+                <span class="sr-only">Next</span>
+            </a>
+        </div>
+        </div>
+    </div>
+            <div class="col-md-3 col-lg-3">
+                <ul class="list-group">
+                    <li class="list-group-item">Lorem ipsum dolor sit amet, eu pro tota</li>
+                    <li class="list-group-item">Lorem ipsum dolor sit amet, eu pro tota</li>
+                    <li class="list-group-item">Lorem ipsum dolor sit amet, eu pro tota</li>
+                </ul>
+            </div>
+        <div class="col-md-3 col-lg3">
```

Comment:
to miało być osobną sekcją (sekcja pod sekcją; tak jak sekcja z zakładkami html, css, javascript, react)